.github/workflows/test-js.yaml

```diff
@@ -12,7 +12,7 @@ concurrency:
   cancel-in-progress: true
 
 env:
-  SUI_VERSION: mainnet-v1.25.3
+  SUI_VERSION: mainnet-v1.31.1
```

Comment:
can we read this from a single place to avoid duplication? Might be useful to add a warning in hardhat.config.js as well if a different sui version is being used locally

Reply:
good idea, but definitely not in the hardhat file 😅